pkg/capture/file/capture_filename.go

```diff
@@ -0,0 +1,18 @@
+package file
+
+import (
+	"fmt"
+	"strings"
+)
+
+type CaptureFilename struct {
+	CaptureName    string
+	NodeHostname   string
+	StartTimestamp *Timestamp
+}
+
+func (cf *CaptureFilename) GenerateCaptureFileName() string {
```

Comment:
This can also likely be a `fmt.Stringer` (see other comment)
```suggestion
func (cf *CaptureFilename) String() string {
```